packages/ibm-products-styles/src/components/SidePanel/_side-panel.scss

```diff
@@ -343,30 +326,29 @@ $action-set-block-class: #{c4p-settings.$pkg-prefix}--action-set;
   }
 
   .#{c4p-settings.$carbon-prefix}--btn.#{$block-class}__navigation-back-button {
-    position: fixed;
+    position: relative;
     z-index: 5;
-    top: 0;
-    left: 0;
+    top: calc(-1 * #{$spacing-03});
+    left: calc(-1 * #{$spacing-03});
```

Comment:
![image](https://github.com/carbon-design-system/ibm-products/assets/1717398/560cf485-0ae6-4271-b9ef-6b6568203287)

Looks like the change in position to be 8px from the end is intentional since I see this consistently with the close button, but looks like we still need to align the icon.